config/routes.rb

```diff
@@ -43,6 +43,7 @@
   get 'choose_registration', to: 'welcome#choose_registration'
   get 'goodbye', to: 'welcome#goodbye'
   get 'bike_creation_graph', to: 'welcome#bike_creation_graph'
+  get "recovery_stories", to: "welcome#recovery_stories", as: :recovery_stories
```

Comment:
Just idle curiosity:
Re. hyphens vs. underscores -- turns out that default I'd internalized at some point was [motivated by SEO](https://www.woorank.com/en/blog/underscores-in-urls-why-are-they-not-recommended). (I think I'd prioritize consistency with existing code anyway, though.)

> Google views hyphens in URLs as word separators while an underscore in your url will not be recognized. This means using hyphens makes it much easier for Google to figure out what a page is about. For example, a URL containing the phrase "my_page" would be interpreted as "mypage" instead of "my page".

Reply:
I went looking after our conversation and found that too!

I think this is a place where consistency trumps that, since the controller action (and the named route) uses underscores.

I'm open to figuring out how to put dashes in all the routes somehow, if it was convenient, since the google reference that site links to does say:

>  We recommend that you use hyphens (-) instead of underscores (_) in your URLs.

... though this seems like a low priority